admin/model/product_table.php

```diff
@@ -0,0 +1,96 @@
+<?php
+include("../includes/header.php");
+    ?>
+    <?php
+// Accepts the product_id
+// Returns a single product as an associative array
+ /*function getProduct($product_id) {
+    include('database.php');
+
+    $query = "SELECT * FROM PRODUCT
+              WHERE product_id = $product_id";
+    $result = mysqli_query($connection, $query);
+
+    if($result) {
+        return mysqli_fetch_assoc($result);
+    } else {
+        return "Error: getProduct()";
+    }
+ }
+    */?>
```

Comment:
Please remove :) We don't want to merge commented out code if we don't need to or it doesn't serve a specific purpose

Reply:
whoops! I meant to do a pull request for shanons repository :)